config.go

```diff
@@ -82,6 +83,23 @@ func (c *Config) InitDefault(log *zap.Logger) {
 	case httpClient:
 	default:
 		c.Client = httpClient
+		// As per https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol
+		exporterEnv := "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
+		exporterVal := os.Getenv(exporterEnv)
+		if exporterVal == "" {
+			exporterEnv = "OTEL_EXPORTER_OTLP_PROTOCOL"
+			exporterVal = os.Getenv(exporterEnv)
+		}
+		switch exporterVal {
```

Comment:
One case is missed - `http/json`.

Reply:
What should it do in the http/json case though? As far as I can tell, its not supported by either client.

Reply:
It doesn't seem particularly necessary but I've made it log a slightly different warning in the http/json case. It'll break either way as the client doesn't support the protocol.